eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -40,15 +40,30 @@ parameters:
     default:
       Public:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePath:
```

Comment:
Weren't you naming this plural somehow in the other PRs?

Reply:
Yes.